src/importers/clubhouseCsv/ClubhouseCsvImporter.ts

```diff
@@ -0,0 +1,188 @@
+import { Importer, ImportResult } from '../../types';
+const csv = require('csvtojson');
+
+type ClubhouseStoryType = 'feature' | 'bug' | 'chore';
+
+interface ClubhouseIssueType {
+  id: string;
+  name: string;
+  type: ClubhouseStoryType;
+  requestor: string;
+  owners: string[];
+  description: string;
+  is_completed: boolean;
+  created_at: Date;
+  started_at: Date;
+  updated_at: Date;
+  moved_at: Date;
+  completed_at: Date;
+  estimate: number;
+  external_ticket_count: number;
+  external_tickets: string[];
+  is_blocked: boolean;
+  is_a_blocker: boolean;
+  due_date: Date;
+  labels: string[];
+  epic_labels: string[];
+  tasks: string[];
+  state: string;
+  epic_id: string;
+  epic: string;
+  project_id: string;
+  project: string;
+  iteration_id: string;
+  iteration: string;
+  is_archived: boolean;
+}
+
+const parseBooleanColumn = (item: string) => item == 'TRUE';
+const parseStringArrayColumn = (item: string) =>
+  item.split(';').filter(s => s.length > 0);
+const parseInt = (item: string) => Number.parseInt(item) || 0;
+const parseDate = (item: string, _: any, __: any, row: string[]) => {
+  if (item.length <= 0) return null;
+  // Inoptimal method for finding the timezone UTC offset, but I don't believe we can get the column names
```

Comment:
```suggestion
  // Inoptimal method for finding the timezone UTC offset, we parse it from the UTC offset column in this row each time
```

Making the comment a bit more clear